aiida/cmdline/commands/cmd_code.py

```diff
@@ -175,33 +175,29 @@ def show(code, verbose):
     table.append(['Prepend text', code.get_prepend_text()])
     table.append(['Append text', code.get_append_text()])
 
-    if verbose:
+    if VERDI_LOGGER.level <= LOG_LEVELS['DEBUG']:
```

Comment:
Hmm, this is a downside of the new solution that I didn't consider. Here the old paradigm of having an on/off switch was clearly superior. Now, one would have to look at the implementation to figure out that when passing `--verbosity DEBUG` one gets the number of calculations done with the code. That is not very intuitive. Is there a way we can improve this?

Reply:
While I agree that this makes the if condition a bit longer here, I would like to point out that this occurs only in cases where it is not possible to express the logging logic via individual `.info` or `.debug` calls (which is possible in most cases).

> Now, one would have to look at the implementation to figure out that when passing --verbosity DEBUG one gets the number of calculations done with the code. That is not very intuitive. Is there a way we can improve this?

I'm not sure exactly what you are referring to here. Are you referring to the fact that by no longer having a `--verbose` option, the documentation of this behavior was lost?
If that is the problem, we could simply add the logging behavior to the command itself, i.e. something like
```
Add `-v DEBUG` to get the number of calculations done with the code.
```
 (or try to override the help string of the `--verbose` option, but I would prefer the former).